app/helpers/admin/corporate_information_pages_helper.rb

```diff
@@ -0,0 +1,48 @@
+module Admin::CorporateInformationPagesHelper
+  include Admin::EditionsHelper
+
+  def index_table_title_row(edition)
+    title = edition.title
+    title += " (#{edition.primary_locale})" if edition.non_english_edition?
+    sanitize(title)
+  end
+
+  def index_table_state_row(edition)
+    state = if edition.force_published?
+              "Force published"
+            elsif edition.unpublishing.present?
+              "Unpublished"
+            else
+              admin_edition_state_text(edition)
+            end
```

Comment:
I think withdrawn editions will inadvertently get caught out by this.

Withdrawals are modelled as 'unpublishing' records – so they will have `edition.unpublishing.present?` and therefore be mislabelled as "Unpublished".

But, this does make me think. Should we actually make this change 'upstream' – i.e. update the `admin_edition_state_text` helper method, rather than wrapping more functionality around it?

That helper method is only used in a couple of places, so it shouldn't be too disruptive to change it 🤞🏻

Reply:
It feels like this is a little bit of a different use case since the other method purposely doesn't use Force published or Unpublished as states as they're not in the state machine.

Maybe they should be? It feels like at the very least unpublishing should be imo.